CHANGELOG.unreleased.md

```diff
@@ -18,6 +18,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [New/Storage] When creating a new storage, if an error occurred while selecting a storage type, nothing was displayed (PR [#8250](https://github.com/vatesfr/xen-orchestra/pull/8250))
```

Comment:
What do you think of: `Correctly display error if storage detection failed for HBA or ZFS`?